docs/rfc-identity.md

```diff
@@ -0,0 +1,186 @@
+# RFC Peer-Star-Identity
+
+This is a proposal for an identity management system for Peer-Star, named Peer-Star-Identity. It's built upon standards to provide a good foundation for Peer-Star applications to authenticate and identify users.
+
+*Authors*: André Cruz, João Santos, Pedro Teixeira
+
+1. [Standards & foundations](#standards--foundations)
+   1. [Decentralized identifiers](#decentralized-identifiers-dids)
+   1. [Self-signed Verifiable Claims](#self-signed-verifiable-claims)
+   1. [DID Delegates & devices](#did-delegates--devices)
+   1. [Using DID-Auth to prove control of the DID](#using-did-auth-to-prove-control-of-the-did)
+1. [Proposal: The IdentityManager](#proposal-the-identitymanager)
```

Comment:
@satazor @joaosantos15 @pgte can you document your analysis (in a separate doc) of the current Identity/Auth/Authorization systems you researched (BlockStack, uPort, OriginProtocol, Keybase) to serve as support for this proposal?

Reply:
Specially on their pros and shortcommings. The insights you share with me were super valuable.

Reply:
Will do!

Reply:
@diasdavid [Here is the analysis of the different DID methods](https://github.com/ipfs-shipyard/peer-star/blob/rfc-identity/docs/did-methods-comparison.md), detailing their shortcomings in addressing our [problem statement](https://github.com/ipfs-shipyard/peer-star/issues/16).

Reply:
@joaosantos15 I've added a link to the document in our problem statement.